frontend/packages/pipelines-plugin/src/components/repository/hooks/pac-hook.ts

```diff
@@ -1,8 +1,28 @@
+import * as React from 'react';
 import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
 import { ConfigMapModel } from '@console/internal/models';
-import { ConfigMapKind } from '@console/internal/module/k8s';
+import { ConfigMapKind, k8sList } from '@console/internal/module/k8s';
+import { RepositoryModel } from '@console/pipelines-plugin/src/models';
 import { PAC_INFO } from '../../pac/const';
 import { PIPELINE_NAMESPACE } from '../../pipelines/const';
 
 export const usePacInfo = () =>
   useK8sGet<ConfigMapKind>(ConfigMapModel, PAC_INFO, PIPELINE_NAMESPACE);
+
+export const useRepositoryPresent = (repoURL) => {
+  const [repoAlreadyExists, setRepoAlreadyExists] = React.useState(false);
+
+  React.useEffect(() => {
+    // eslint-disable-next-line promise/catch-or-return
```

Comment:
Please catch and log this error instead of just ignoring it. You might also want `setRepoAlreadyExist` to false again so that the user can submit the form?